tools/flaky-test-retryer/handler.go

```diff
@@ -94,16 +96,16 @@ func (hc *HandlerClient) HandleJob(jd *JobData) {
 	}
 	logWithPrefix(jd, "got %d flaky tests from today's report\n", len(flakyTests))
 
-	if outliers := getNonFlakyTests(failedTests, flakyTests); len(outliers) > 0 {
+	outliers := getNonFlakyTests(failedTests, flakyTests)
+	if len(outliers) > 0 {
```

Comment:
this if condition is done twice, does this logic have to be here?

Reply:
this was so I could log how many tests are not flaky and whether or not we triggered a retry. I could move the logic, or forgo it altogether as it's mostly for testing purposes

Reply:
This would be helpful debugging message, just think it could be combined with the other `if else` if you agree with what I suggested over there

Reply:
done